lab-bundle-spec.md

```diff
@@ -521,16 +521,16 @@ assessment:
           # Check for bucket
           found_bucket = ...
           unless found_bucket
-            return { score: 0, student_message: 'bucket_missing' }
+            return { score: 0, message: 'bucket_missing', student_message: 'bucket_missing' }
```

Comment:
in the examples here and below we should not have the `message` match the `student_message` and look like a localization key

Reply:
What should it look like? `message` does have access to the localization key's message.

Reply:
i am fine going either way. we should either have them be strings that make sense as log messages, or keys that point to log-specific sets of messages. either way we probably want to document in the description of the field how it works.

Reply:
[Updated Spec PR](https://github.com/CloudVLab/qwiklabs-content-bundle-spec/pull/81)